feature-libs/pdf-invoices/karma.conf.js

```diff
@@ -4,9 +4,8 @@
 module.exports = function (config) {
   config.set({
     basePath: '',
-    frameworks: ['parallel', 'jasmine', '@angular-devkit/build-angular'],
+    frameworks: ['jasmine', '@angular-devkit/build-angular'],
     plugins: [
-      require('karma-parallel'),
       require('karma-jasmine'),
```

Comment:
[QUESTION]
Why is removing `karma-parallel` related to the upgrade of rxjs?

btw. I can understand that for small libraries that have little unit tests, the parallelization of the unit tests execution might not bring much value. `pdf-invoices` might be one of such libraries. But I'm wondering why you made this change in _this_ rxjs-upgrad PR ?

Note: analogical question to other files with removed karma-parallel

Reply:
Then, shall we remove `karma-parallel` from all the libraries, or just those that use `jasmine-marbles`?
Just to better understand the consequences: What's the impact of removing `karma-parallel` from all libraries on the overall time waiting for CI to pass?

Reply:
Here's the separate ticket for this: https://jira.tools.sap/browse/CXSPA-4869